TKGame/Level Editor Content/Stage.cs

```diff
@@ -85,6 +86,17 @@ public void Update()
             }   
         }
 
+        public void Draw(SpriteBatch spriteBatch)
+        {
+            //Draws the image into the Background
+            if (stageName == "room0.json")
+                spriteBatch.Draw(Art.BackgroundTexture1, Background.BackgroundRect, Color.White);
+            else if (stageName == "room1.json")
+                spriteBatch.Draw(Art.BackgroundTexture2, Background.BackgroundRect, Color.White);
+            else
+                spriteBatch.Draw(Art.BackgroundTexture3, Background.BackgroundRect, Color.White);
+        }
+
```

Comment:
This is NOT what I meant. When a stage is initialized it should have a Background variable that gets drawn in ` spriteBatch.Draw(**this.Background**, Background.BackgroundRect, Color.White);`. As it stands this would require us to add another else if for EVERY stage and EVERY background and hard code them to different rooms. I don't want to have to refactor more things when we add procedural level generation and this would be just another thing in that list.